virtualizarr/writers/icechunk.py

```diff
@@ -51,7 +53,10 @@ def dataset_to_icechunk(ds: Dataset, store: "IcechunkStore") -> None:
 
     # TODO only supports writing to the root group currently
     # TODO pass zarr_format kwarg?
-    root_group = Group.from_store(store=store)
+    if store.mode.str == "a":
+        root_group = Group.open(store=store, zarr_format=3)
+    else:
```

Comment:
Is `zarr_format=3` needed for `from_store` and `open` or just open?

Reply:
If it wasn't needed before I'm not sure why it would be needed now